net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -78,6 +78,10 @@ function quagga_ospfd_install_conf() {
 	// generate ospfd.conf based on the assistant
 	if (is_array($config['installedpackages']['quaggaospfd']['config'])) {
 		$ospfd_conf = &$config['installedpackages']['quaggaospfd']['config'][0];
+	} elseif (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospfd'])  ||
+		  isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d']) ||
+	 	  isset($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
```

Comment:
As defined in Developer style guide [1], if a conditional statement must span multiple lines it should use 4 spaces indent:
```
if (foo) {
        ...
} elseif ($foo1 && $foo2 && $foo3 && $foo4 && $foo5 && $foo6 &&
    $foo7 && foo8 && foo9) {
        ...
}
```

[1] https://doc.pfsense.org/index.php/Developer_Style_Guide